examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala

```diff
@@ -30,14 +25,14 @@ object LiveJournalPageRank {
   def main(args: Array[String]) {
     if (args.length < 1) {
       System.err.println(
-        "Usage: LiveJournalPageRank <edge_list_file>\n" +
+        "Usage: LiveJournalPageRank <edge_list_file> --numEPart=<num_edge_partitions>\n" +
           "    [--tol=<tolerance>]\n" +
           "        The tolerance allowed at convergence (smaller => more accurate). Default is " +
           "0.001.\n" +
           "    [--output=<output_file>]\n" +
           "        If specified, the file to write the ranks to.\n" +
           "    [--numEPart=<num_edge_partitions>]\n" +
-          "        The number of partitions for the graph's edge RDD. Default is 4.\n" +
+          "        The number of partitions for the graph's edge RDD.\n" +
```

Comment:
Eh, but now you have put the option in twice! why not just move it up and remove the square braces to mark it non-optional? keep everything else the same.